reflex-dom-plaid/Reflex/Dom/Plaid/Link.hs

```diff
@@ -131,21 +131,31 @@ activatePlaidLinkDialog cfg onResult = do
     pure ()
     )
 
+  o ^. jss (t_ "onEvent") (fun $ \_ _ args -> case args of
+      (eventNameJSVal : metaJs : _) -> do
+        eventName <- filter (/= '"') . show <$> fromJSValUnchecked @JSString eventNameJSVal
```

Comment:
Can this not be `fromJSVal @Text eventNameJSVal`?

Reply:
You suggestion worked. Should I keep `fromJSValUnchecked ` or use more safe `fromJSVal`?

Reply:
`fromJSVal @Text eventNameJSVal` is considerably cleaner.